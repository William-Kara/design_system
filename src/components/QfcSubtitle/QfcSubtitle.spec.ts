import { it, describe, beforeEach, expect } from 'vitest';
import { shallowMount } from '@vue/test-utils';
import Subtitle from './QfcSubtitle.vue';

let wrapper: any = null;

const propsValues = {
  text: 'Subtitle',
};

beforeEach(() => {
  wrapper = shallowMount(Subtitle, {
    propsData: propsValues,
  });
});

describe('Subtitle Component', () => {
  it('should mount properly', () => {
    expect(wrapper).toBeTruthy();
  });

  it('should have correct props', () => {
    expect(wrapper.props()).toEqual(propsValues);
  });

  it('should render correctly', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
