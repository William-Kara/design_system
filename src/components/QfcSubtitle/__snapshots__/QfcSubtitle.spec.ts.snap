// Vitest Snapshot v1

exports[`Subtitle Component > should render correctly 1`] = `"<div data-v-038822c7=\\"\\" class=\\"qfc-subtitle\\">Subtitle</div>"`;
